src/openzeppelin/tests/token/test_erc721.cairo

```diff
@@ -474,9 +534,14 @@ fn test_transfer_from_to_owner() {
     assert(ERC721Impl::owner_of(@state, TOKEN_ID) == OWNER(), 'Ownership before');
     assert(ERC721Impl::balance_of(@state, OWNER()) == 1, 'Balance of owner before');
 
-    set_caller_address(OWNER());
+    testing::set_caller_address(OWNER());
     ERC721Impl::transfer_from(ref state, OWNER(), OWNER(), TOKEN_ID);
 
+    let event = testing::pop_log::<Transfer>(ZERO()).unwrap();
+    assert(event.from == OWNER(), 'Invalid from');
+    assert(event.to == OWNER(), 'Invalid to');
+    assert(event.token_id == TOKEN_ID, 'Invalid token_id');
+
```

Comment:
For clarity, a helper like this:

```
fn assert_event_Approval_For_All(owner: ContractAddress, operator: ContractAddress, approved: bool) {
    let event = testing::pop_log::<ApprovalForAll>(ZERO()).unwrap();
    assert(event.owner == owner, 'Invalid owner');
    assert(event.operator == operator, 'Invalid operator');
    assert(event.approved == approved, 'Invalid approved');
}
```

Reply:
There's a lot indeed, I like the helper idea.

Reply:
mmm, for some reason I've pushed to the branch successfully but the PR is not updated. Seems like GitHub is having some issues. Hopefully, this will be fixed automatically later.